### ISSUE TITLE
Decide on how to support templating so contributions are "easy" or "low barrier" 

### ISSUE THREAD

**Issue by destos** [tulsawebdevs/tulsawebdevs.org#3]
It would be nice if users that aren't familiar with Django to be able to contribute to the look and functionality of the frontend of the website.

This can probably be accomplished in a few ways:

* a virtual container with content fixtures. ( Docker, Vagrant )
* separating the template system into it's own module system with mocked data.
 * by supporting loading templates as module packs, allows eventual expansion to support multiple user groups
* keep all templates in same repo and user has to run Django manually
* …probably a few more, add your opinion?

They will still most likely still have to learn Jinja2 and whatever frontend build process we settle on. ( unless that is the responsibility of the template pack )

**Comment by buddylindsey** [comment]
Devils advocate: Is this really worth the effort? How often are people going to want to tweak on TWD site itself?

**Comment by wassimk** [comment]
I agree with @buddylindsey. I've find it easier in the long run to bring people up instead of taking the work down to their level. As a non-Django developer I think it would be cool if there was a beginner talk to introduce Django and goes over this app.

**Comment by destos** [comment]
I was planning to give a talk at the next meeting about progress and decisions made thus far.

**Comment by destos** [comment]
@wassimk, @buddylindsey is giving and intro to Django talk at Tulsa Tech Fest on Friday!
http://techfests.com/Tulsa/2015/Tracks/Miscellaneous/default.aspx
